stream2segment/download/modules/events.py

```diff
@@ -374,26 +343,6 @@ def _get_freq_mag_distrib(evt_query_args):
     return minmag, step, ret
 
 
-class ISFParseError(Exception):
-    pass
-
-
-def isfresponse2txt(nonempty_text, catalog='ISC', contributor='ISC'):
-    """Convert an isf formatted string into an FDSN text formatted string
-    Expects text to be non empty, raises if no event could be parsed
-    """
-    sio = StringIO(nonempty_text)
-    sio.seek(0)
-    try:
-        ret = '\n'.join('|'.join(_) for _ in isf2text_iter(sio, catalog, contributor))
-        # if text is non empty, we can safely raise unformatted type:
-        if not ret:
-            raise Exception('Event block not found')  # caught and reraised below
-        return ret
-    except Exception as exc:  # pylint: disable=broad-except
-        raise ISFParseError(str(exc))
-
-
 def isf2text_iter(isf_filep, catalog='', contributor=''):
```

Comment:
I think this function can be removed, too, as well as all potential functions called only by this function